doc/content/reference/components/network-operations-center.md

```diff
@@ -11,6 +11,109 @@ Network Operations Center provides aggregated insight in a network operated with
 
 Network Operations Center stores events from {{% tts %}} in a time series database, queries aggregated data and visualizes data in a dashboard driven by Grafana. Network Operations Center is complementary to the [Console]({{< relref "console" >}}): the Network Operations Center shows historic data (with a delay of at most 5 minutes), while the Console shows live information and allows for managing the network.
 
-{{< figure src="../network-operations-center_overview.png" alt="Network Operations Center" >}}
+## Accessing Network Operations Center
+
+The Network Operations Center can only be accessed by administrators. There are two main ways to access Network Operations Center within {{% tts %}} console:
+
+- On the vertical navigation click Network Operations Center.
+- On the horizontal navigation click on your avatar/profile name and then from the drop-down menu click Network Operations Center.
+
+The Network Operations Center Overview dashboard will open in a new web browser tab.
+
+{{< note "Network Operations Center is supported in the current version of these [web browsers](https://grafana.com/docs/grafana/latest/setup-grafana/installation/)." />}}
+
+A dashboard provides many features that can be used to customize the presentation of data. The following image highlights some of them.
+
+{{< figure src="../dashboard-features.png" alt="dashboard features" >}}
+
+Horizontal navigation bar:
+
+- **Dashboard title (1)**: Displays the name of the dashboard. When you click on it you will redirect to the Search dashboards by name page.
+- **Share dashboard (2)**: To share a dashboard or panel as a link, snapshot, or JSON file.
+- **Time picker (3)**: To select relative time range options and set custom absolute time ranges. For more information on using the time picker read the Grafana [Time range controls](https://grafana.com/docs/grafana/v9.0/dashboards/time-range-controls/#common-time-range-controls) documentation.
+- **Refresh dashboard (4)**: To refresh the dashboard data.
+- **Refresh dashboard time interval (5)**: To select a dashboard auto-refresh time interval.
+- **Zoom out time range (6)**: To view a larger time range in the dashboard or panel visualization.
+- **View mode (7)**: To display the dashboard on a large screen such as a TV or a kiosk. Press ESC to return to normal mode.
+- **Console (8)**: To open {{% tts %}} console.
+
+Vertical navigation bar:
+
+- **Grafana logo (9)**: To redirect to the Network Operations Center Overview dashboard (home dashboard).
+- **Search dashboards (10)**: To search for dashboards by name. Currently, there are two dashboards available in Network Operations Center: Overview and Gateway Details.
+- **Dashboards (11)**: To manage dashboards and folders 
+- **Alerting (12)**: To set alert rules and notifications
```

Comment:
This looks messy. I think we don't really need these numbers on this image, and some of the explanations are redundant (because they're too obvious or have nothing to do with TTS). We don't need to explain what happens when you click on Grafana logo, or when you click on on Refresh dashboard. There's Grafana documentation for those things. You can say something like "Grafana offers many features such as time picker, alerting, etc. For more info see the Grafana docs." and link the official Grafana documentation.